svg-battery-indicator.el

```diff
@@ -89,7 +94,7 @@ CHARGING is non-nil a lightning symbol is drawn over the SVG."
 		      'battery-load-critical)
 		     ((<= percentage battery-load-low)
 		      'battery-load-low)
-		     (t 'mode-line))
```

Comment:
I like this idea, but I'm not sure we need to use face.

I used faces for the other colours because I wanted automatic consistency with the standard battery colours, and those are setup as faces. But here, there isn't anything obvious to inherit from. Or in other words: from the users point of view, this is isn't the sort that is usually customised by changing a face. Maybe we should just use a colour value string?